.eslintignore

```diff
@@ -0,0 +1,3 @@
+**/node_modules/*
+**/out/*
+**/.next/*
```

Comment:
Ideally, we are only going to have one `node_modules`, `.out`, and `.next` directories so there is no need for stars. and always try to sort such config files ;D
```suggestion
.next
node_modules
out
```